platform-cloud/docs/compute-envs/aws-cloud.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "AWS Cloud VM"
+description: "Instructions to set up an AWS Cloud VM in Seqera Platform"
+date: "12 May 2025"
+tags: [cloud, vm, amazon, compute environment]
+---
+
+::: note
+This guide assumes you already have an AWS account with a valid AWS subscription.
+::: 
+
+The current implementation of compute environments for cloud providers all rely on the use of batch services such as AWS Batch, Azure Batch, and Google Batch for the execution of submitted jobs including the Nextflow application running Studio environments. Batch services work well for large scale workloads, however they add a lot of complexity and overhead to run a few compute jobs. In practical terms the current use of batch services results in limitations such as:
+
+- **Long launch delay**: When launching a pipeline execution, there's a delay of several minutes before the pipeline starts running or the Studio environment is launched. This is caused by the batch services that need to provision the associated compute service to run even a single job.
+- **Complex setup**: Services such as AWS Batch requires complex IAM policies and the configuration of complex services such as Batch Compute Environment, Batch Job definition, Batch queues, etc.
+- **Allocation constraints**: Services such as AWS Batch have strict quotas on the number of resources, for example the number of queues that can be created has a hard limit of 50. This means that no compute environment can be created beyond that limit.
+
+The AWS Cloud VM compute environment addresses these pain points with:
+
+- **Faster startup time**: Nextflow pipelines reach a `Running` status and Studio sessions connect in under a minute (a 4x improvement compared to classic AWS Batch compute environment).
+- **Simplified configuration**: Leaner configurable options, with opinionated defaults to provide the best Nextflow pipeline execution environment, with Wave and Fusion enabled.
+- **Fewer AWS dependencies**: Only one IAM role in AWS is needed, which is subject to a 1000 soft quota limit per Region.
+- **Spot instances**: Studios can be launched on a Spot instance.
+
+This compute environment is best suited to run Studios and small to medium sized pipelines and offers more predictable compute pricing given the fixed instance type. It spins up a standalone, On-Demand EC2 instance, and executes a Nextflow pipeline with a local executor on the EC2 machine. At the end of the execution, the instance is terminated.
```

Comment:
Aftern Endre's change is no longer true that it will always spin up On-Demand instances, (i.e. users could choose Spot for Studios) so it would probably best to remove the On-Demand word, wdyt?